tests/zombienet/setup-preconditions-api.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const { Keyring } = require('@polkadot/keyring');
+const util = require('@polkadot/util');
+const TEST_DAPP_ADDRESS = '0x0000000000000000000000000000000000000001';
+
+async function run(nodeName, networkInfo, args) {
+  const BN = require('bn.js');
+  const ONE = new BN(10).pow(new BN(18));
+
+  const { sendTransaction } = await import('./tx-utils.mjs');
+  const { wsUri, userDefinedTypes } = networkInfo.nodesByName[nodeName];
+  const provider = new WsProvider(wsUri);
+  const api = await ApiPromise.create({ provider });
+
+  await api.isReady;
+
+  // account to submit tx
+  const keyring = new Keyring({ type: 'sr25519' });
+  const sender = keyring.addFromUri('//' + args[0]);
+
+  // create asset
+  console.log('Creating asset with sender: ', sender.address);
```

Comment:
How about using `console.info` instead of `console.log` for those we want to have it (remain in the repo)
CC: @bobo-k2 @gluneau

Reply:
Cleaned up at this [commit](https://github.com/AstarNetwork/astar-apps/pull/789/commits/a277bbec7d544ec3a31ff27fe2f26edd1fafe517)